sections/expansion_content.tex

```diff
@@ -51,66 +75,189 @@ \subsection*{Empowered Statistic Cards}
   \medskip
 
   \begin{minipage}[h]{\linewidth}
-    \vspace{0.1pt}
-    \centering
-    \begin{scriptsize}
-      \begin{tikzpicture}
-        \draw (0, 0) node[inner sep=0] {\makebox[\linewidth][c]{\includegraphics[width=\linewidth]{\cards/event.png}}};
-        \draw (1, 1.8) node {\encircle{\phantom{.}1\phantom{.}}};
-        \draw (-2, 0.4) node {\encircle{\phantom{.}2\phantom{.}}};
-        \draw (2.5, -1.1) node {\encircle{\phantom{.}3\phantom{.}}};
-      \end{tikzpicture}
-    \end{scriptsize}
-    \footnotesize
-    \textbf{\textit{\textcolor{darkcandyapplered}{Event Card}}}
-    \begin{multicols}{2}
-      \begin{itemize}
-        \item[\textbf{1.}] Name
-        \item[\textbf{2.}] Fluff
-        \item[\textbf{3.}] Effect
-        \item[\textbf{\phantom{.}}] \phantom{.}
-      \end{itemize}
-    \end{multicols}
+  \vspace{0.1pt}
+  \centering
+  \begin{scriptsize}
+    \begin{tikzpicture}
+    \draw (0, 0) node[inner sep=0] {\makebox[\linewidth][c]{\includegraphics[width=\linewidth]{\cards/event.png}}};
+    \draw (1, 1.8) node {\encircle{\phantom{.}1\phantom{.}}};
+    \draw (-2, 0.4) node {\encircle{\phantom{.}2\phantom{.}}};
+    \draw (2.5, -1.1) node {\encircle{\phantom{.}3\phantom{.}}};
+    \end{tikzpicture}
+  \end{scriptsize}
+  \footnotesize
+  \textbf{\textit{\textcolor{darkcandyapplered}{Event Card}}}
+  \begin{multicols}{2}
+    \begin{itemize}
+    \item[\textbf{1.}] Name
+    \item[\textbf{2.}] Fluff
```

Comment:
`Fluff` -> `Flavor Text`

Reply:
Fluff is actually taken straight from the original. You think flavor text is better?

<img width="752" height="300" alt="image" src="https://github.com/user-attachments/assets/f6a07a77-6d6d-4f85-8436-692d81a46eb4" />

Reply:
Flavor text is the common term for this thing in other games

Reply:
I noticed that translations tend to prefer it too. Shall we change it everywhere?

Reply:
It's only in one other place. I'll change it.